### PR TITLE
Define public FulfillRequisition method for direct Kingdom measurements

### PR DESCRIPTION
**Pull request by renjiezh**
[Design Doc](https://docs.google.com/document/d/1mgGoTqVUUs8zDcOOmiRECV10pXPd1TOz2CoLKqOpTDI/edit#)

---
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)

### CODE REVIEW

File: src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto [hunk 1]
```diff
@@ -90,3 +94,30 @@ message RefuseRequisitionRequest {
...
   Requisition.Refusal refusal = 2;
 }
+
+// Request message for `FulfillRequisition` method.
+message FulfillRequisitionRequest {
+  // Resource name of the `Requisition. Required.
+  string name = 1
+      [(google.api.resource_reference).type = "halo.wfanet.org/Requisition"];
+
+  // Encrypted `SignedData` containing the data for fulfillment. Required.
```
Comment (tristanvuong2021) [inline src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto:104]: I realized that there should be additional info here in the comment to clarify. Something like
  //
  // The encryption uses the `measurement_public_key` from `measurement_spec` as
  // the recipient public key.

**Review by tristanvuong2021 — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @renjiezh and @tristanvuong2021)

**Review by tristanvuong2021 — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @renjiezh and @tristanvuong2021)

___
*[src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto, line 104 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91#-MxgBG-ExtY6YnZaT_wX-r1-104:-MxjmabN7XRD_PdX5Dg9:bv6tin8) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement-api/blob/7955016134ecc8e3f89f50ef4495d6cea34f42ef/src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto#L104)):*
<details><summary><i>Previously, tristanvuong2021 (Tristan Vuong) wrote…</i></summary><blockquote>

I realized that there should be additional info here in the comment to clarify. Something like
  //
  // The encryption uses the `measurement_public_key` from `measurement_spec` as
  // the recipient public key.
</blockquote></details>

// The encryption uses the `measurement_public_key` from `measurement_spec` as
// the recipient public key.

**Review by tristanvuong2021 — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @renjiezh and @tristanvuong2021)

___
*[src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto, line 104 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91#-MxgBG-ExtY6YnZaT_wX-r1-104:-MxjmvpFDayp3rgHrKK2:undefined) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement-api/blob/7955016134ecc8e3f89f50ef4495d6cea34f42ef/src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto#L104)):*
<details><summary><i>Previously, tristanvuong2021 (Tristan Vuong) wrote…</i></summary><blockquote>

// The encryption uses the `measurement_public_key` from `measurement_spec` as
// the recipient public key.
</blockquote></details>





_Code snippet:_
```protobuf
// The encryption uses the `measurement_public_key` from `measurement_spec` as
// the recipient public key.
```

**Review by renjiezh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @tristanvuong2021)

___
*[src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto, line 104 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91#-MxgBG-ExtY6YnZaT_wX-r1-104:-Mxjz-2m6p20BX8cmmsy:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement-api/blob/7955016134ecc8e3f89f50ef4495d6cea34f42ef/src/main/proto/wfa/measurement/api/v2alpha/requisitions_service.proto#L104)):*
> ```protobuf
>       [(google.api.resource_reference).type = "halo.wfanet.org/Requisition"];
> 
>   // Encrypted `SignedData` containing the data for fulfillment. Required.
> ```

Done.

**Review by tristanvuong2021 — APPROVED** [review]

**Review by tristanvuong2021 — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @tristanvuong2021)

**Review by stevenwarejones — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @renjiezh)

**Review by renjiezh — COMMENTED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @renjiezh)

**Review by renjiezh — COMMENTED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement-api/91)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @renjiezh)

**Pull request merged**